eng/common/pipelines/templates/steps/get-code-owners.yml

```diff
@@ -0,0 +1,25 @@
+parameters: 
+  TargetUserVariable: 'notspecified'
+  TargetTeamVariable: 'notspecified'
+  TargetLabelVariable: 'notspecified'
+  ServiceDirectory: ''
+  DevOpsFeed: "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json"
+
+steps:
+  - task: DotNetCoreCLI@2
+    displayName: 'Install CodeOwners Retriever'
+    inputs:
+      command: custom
+      custom: 'tool'
+      arguments: 'install --tool-path . --add-source "${{ parameters.DevOpsFeed }}" --version "1.0.0-dev.20211019.1" "Azure.Sdk.Tools.RetrieveCodeOwners"'
+      workingDirectory: '$(Agent.BuildDirectory)'
```

Comment:
I'm not sure if using the BuildDirectory is the best option here as it might cause conflicts with other things in the repo clone. I'd suggest `Agent.ToolsDirectory`

Reply:
Done. And will test it